policyengine_us/reforms/co_general_assembly/co_family_affordability_credit.py

```diff
@@ -0,0 +1,89 @@
+from policyengine_us.model_api import *
+
+
+def create_co_family_affordability_credit() -> Reform:
+    class co_family_affordability_credit(Variable):
+        value_type = float
+        entity = Person
+        label = "Colorado Family Affordability Credit"
+        unit = USD
+        definition_period = YEAR
+        reference = "https://leg.colorado.gov/bills/hb24-1311"
+        defined_for = StateCode.CO
+
+        def formula(person, period, parameters):
+            age = person("age", period)
+            depenent = person("is_child_dependent", period)
+            p = parameters(
+                period
+            ).gov.contrib.co_general_assembly.family_affordability_credit
+            base_amount = p.amount.calc(age) * depenent
+            agi = person.tax_unit("adjusted_gross_income", period)
+            filing_status = person.tax_unit("filing_status", period)
+            phase_out_start = where(
```

Comment:
please add comments to explain this non-obvious code

Reply:
added

Reply:
Oh I see, the first boundary is the young child age limit. Please create a new python variable for it to clarify and reduce duplication.

Reply:
done